src/store/modules/Health/EventLogStore.js

```diff
@@ -0,0 +1,68 @@
+import api from '../../api';
+
+const EventLogStore = {
+  namespaced: true,
+  state: {
+    eventLogData: null
+  },
+  getters: {
+    eventLogData(state) {
+      return state.eventLogData;
+    }
+  },
+  mutations: {
+    setEventLogData(state, eventLogData) {
+      state.eventLogData = eventLogData;
+    }
+  },
+  actions: {
+    getEventLogData({ commit }) {
+      api
+        .get('/xyz/openbmc_project/logging/enumerate')
+        .then(({ data }) => {
+          const eventLog = data.data;
```

Comment:
Are we going to use object destructuring or maintain consistency in showing the response's data structure? I feel like its more readable when not using object destructuring.